cfme/web_ui/form_buttons.py

```diff
@@ -28,13 +28,15 @@ class FormButton(Pretty):
 
     class Button:
         """Holds pieces of the XPath to be assembled."""
-        TAG_TYPES = "//a | //button | //img | //input"
+        TAG_TYPES = "//a | //button | //img | //input | //miq-button"
         TYPE_CONDITION = (
-            "(contains(@class, 'button') or contains(@class, 'btn') or contains(@src, 'button'))"
+            "(contains(@class, 'button') or contains(@class, 'btn') or contains(@src, 'button') "
+            "or contains(@class,'ng-isolate-scope'))"
         )
         DIMMED = "(contains(@class, 'dimmed') " \
             "or contains(@class, 'disabled') " \
-            "or contains(@class, 'btn-disabled'))"
+            "or contains(@class, 'btn-disabled')" \
+            "or contains(@class, 'ng-isolate-scope ng-hide'))"
```

Comment:
This is not correct - at least not for Infra Provider Add page. `.is_dimmed` check doesn't work because of that.